src/components/AttentionBox/AttentionBox.module.scss

```diff
@@ -6,7 +6,7 @@
   position: relative;
   padding: var(--spacing-medium);
   border-radius: var(--border-radius-medium);
-  width: fit-content;
+  width: 100%;
```

Comment:
dangerous a bit (breaking change)

Reply:
but we said that we are intentionally want it 👀

Reply:
will try to solve with fit-content 👍🏼